lib/msal-node/test/client/PublicClientApplication.spec.ts

```diff
@@ -1,8 +1,12 @@
 import { PublicClientApplication } from './../../src/client/PublicClientApplication';
 import {AuthorizationCodeRequest, ClientConfiguration} from './../../src/index';
-import { DeviceCodeRequest } from "@azure/msal-common/dist/src/request/DeviceCodeRequest";
-import {AuthorizationCodeClient, AuthorizationCodeUrlRequest, DeviceCodeClient} from "@azure/msal-common";
 import {AUTHENTICATION_RESULT, TEST_CONSTANTS} from "../utils/TestConstants";
+import {
+    AuthorizationCodeClient,
+    AuthorizationCodeUrlRequest,
+    DeviceCodeClient,
+    DeviceCodeRequest,
+    RefreshTokenRequest} from "@azure/msal-common";
```

Comment:
Can put the "from" part on the next line.

Reply:
Done